Octokit.Tests/Clients/BlobClientTests.cs

```diff
@@ -122,7 +122,7 @@ public void CanDeserializeBlob()
             Assert.NotNull(response.Body);
             Assert.Equal(blobResponseJson, (string)response.HttpResponse.Body);
             Assert.Equal(100, response.Body.Size);
-            Assert.Equal(EncodingType.Utf8, response.Body.Encoding);
+            Assert.Equal(EncodingType.Utf8, response.Body.Encoding.ParsedValue);
```

Comment:
> Although more work to change 42 enums, I think Im keen on my earlier suggestion of being explicit with [Parameter()] attributes on every enum member, enforced via convention test, and relied on by the deserializer

I agree. Explicit is better. This lets us simplify the serializer as well. Want to check in the convention test? I can go through and add attributes.